server/migrations/1643746660201_create-participant-post-hire-status.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable camelcase */
+const { dbClient, schema, collections } = require('../db');
+
+exports.shorthands = undefined;
+
+exports.up = async () => {
+  await dbClient.runRawQuery(schema.postHireRelationTables[0].definition);
```

Comment:
Instead of running a raw query, it would be better practice to use the query builder like we did for the `capture-waitlist-emails` migration

Reply:
Updated please check